doc/guide/emacs.md

```diff
@@ -32,6 +32,17 @@ And you can now run Gerbil with `M-x run-scheme`.
 Note that both `gerbil-mode.el` and `gambit.el` are installed by
 default in `$GERBIL_INSTAL_PREFIX/share/emacs/site-lisp` when installing gerbil.
 
+## SLIME Integration
+
+There's a `SLIME: The Superior Lisp Interaction Mode for Emacs` server that provides a near IDE experience for gerbil inside emacs.
+
+This is very useful for connecting emacs to a gerbil instance and controlling it that way. 
+
+Until there's more detail in this documentation have a look at the README in the ide/swank directory of the [Gerbil Source Tree](https://github.com/mighty-gerbils/gerbil/tree/master/src/std). 
+
```

Comment:
the link points to std... not useful at all. Let's please add some basic instructions on how to use it.

Reply:
At the very least point to the swank README.org

Reply:
Ideally add that documentation here.